lib/matplotlib/patheffects.py

```diff
@@ -359,3 +361,164 @@ def draw_path(self, renderer, gc, tpath, affine, rgbFace):
         if clip_path:
             self.patch.set_clip_path(*clip_path)
         self.patch.draw(renderer)
+
+
+class TickedStroke(AbstractPathEffect):
+    """A line based :class:`PathEffect` which draws a path with a
```

Comment:
```suggestion
    """A line based PathEffect which draws a path with a
```
We don't have a `PathEffect` class, technically, that would be `AbstractPathEffect`. But you can leave it plain text like in the other classes.

Also please start and end multi-line docstrings on a separate line, i.e.

~~~
"""
A line based PathEffect ...

a line pattern.
"""